plugin/wavesurfer.regions.js

```diff
@@ -331,6 +331,9 @@ WaveSurfer.Region = {
             my.element.addEventListener('mousedown', onDown);
             my.wrapper.addEventListener('mouseup', onUp);
             my.wrapper.addEventListener('mousemove', onMove);
+            document.querySelector("body").addEventListener("mouseup", function () {
```

Comment:
`document.body.addEventListener('mouseup', onUp)` — shorter, the same meaning.

Please, also use single quotes.